### ISSUE TITLE
Scan not using correct path in NC19?

### ISSUE THREAD

**Issue by pjft** [Blaok/nextcloud-inotifyscan#17]
Hi! Thanks for putting this together.

There seems to be something off since I upgraded to NC19. I just looked into my files and they were consistently not being scanned, even though they're detected as changed. I went through the logs, and it seems that the command is being invoked with a missing "/".

Here are the logs:

```
Sep  5 12:59:02 openmediavault nextcloud-inotifyscan[2801]: INFO - watching /srv/dev-disk-by-label-HDD/nextcloud/admin/files <=> /admin/files
Sep  5 12:59:02 openmediavault nextcloud-inotifyscan[2801]: INFO - watching /srv/dev-disk-by-label-HDD/root/ <=> /admin/files/Root
Sep  5 12:59:02 openmediavault nextcloud-inotifyscan[2801]: INFO - watching /srv/dev-disk-by-label-HDD/notes/ <=> /admin/files/Notes
Sep  5 12:59:02 openmediavault nextcloud-inotifyscan[2801]: INFO - watching /srv/dev-disk-by-label-HDD/other/ <=> /admin/files/Other
Sep  5 13:00:01 openmediavault CRON[3027]: (www-data) CMD (php -f /var/www/html/nextcloud/cron.php)
Sep  5 13:00:01 openmediavault CRON[3028]: (pi) CMD (sudo /sbin/folder2ram -sync 4)
Sep  5 13:02:55 openmediavault systemd[1]: Started Session c3 of user pi.
Sep  5 13:03:49 openmediavault nextcloud-inotifyscan[2801]: INFO - Found /srv/dev-disk-by-label-HDD/root/myfolder/ CREATE test6.txt
Sep  5 13:03:49 openmediavault nextcloud-inotifyscan[2801]: INFO - Scan for /admin/files/Rootmyfolder/test6.txt
```

If you see, in the last one it seems to miss a "/" between the admin/files/Root external storage and the myfolder folder. 

The external storages have a "/" at the end in the Nextcloud settings.

In the Nextcloud database it also has the end "/" so it seems to be something in the script, it seems?

```
MariaDB [nextcloud]> select * from oc_storages limit 10;
+------------+----------------------------------------------+-----------+--------------+
| numeric_id | id                                           | available | last_checked |
+------------+----------------------------------------------+-----------+--------------+
|          2 | home::admin                                  |         1 |         NULL |
|         28 | local::/sharedfolders/root/nextcloud/        |         1 |         NULL |
|         29 | local::/srv/dev-disk-by-label-HDD/nextcloud/ |         1 |         NULL |
|         31 | local::/srv/dev-disk-by-label-HDD/root/      |         1 |   1599307516 |
|         32 | local::/srv/dev-disk-by-label-HDD/notes/     |         1 |   1599307516 |
|         33 | local::/srv/dev-disk-by-label-HDD/other/     |         1 |   1599307516 |
|         34 | local::/var/www/html/nextcloud/data/         |         1 |         NULL |
+------------+----------------------------------------------+-----------+--------------+
```

I'm happy to dig into it further, but I'm a bit at a loss, so any pointers would be appreciated. Thank you.

**Comment by pjft** [comment]
In my case I address it by appending a "/" in line 191, but it might be missing similar logic for non-external-storage folders, as the 
`/admin/files` path still shows without the last '/'.

**Comment by Blaok** [comment]
Thanks for the bug report. The problem was caused by inconsistency between the “data prefix” and the “path prefix”, which affects external storage only. I'll amend and merge your pull request shortly. Details about the problem will be left in the pull request for clearer context.

**Comment by Blaok** [comment]
Fixed by #18

**Comment by pjft** [comment]
Thanks - and apologies for the incomplete attempt at solving it on my end!
